src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -67,53 +67,6 @@
 #endif
 #endif
 
-/* RMA Key Space division
```

Comment:
What does the key contain now?

Reply:
Now is just a unique number. It does not have anything like key type or context ID.